packages/itwin/grouping-mapping-widget/src/components/Groups/QueryBuilder/QueryBuilder.ts

```diff
@@ -219,6 +237,7 @@ export class QueryBuilder {
     propertyValue: Primitives.Value,
     needsQuote: boolean,
     isCategory: boolean,
+    modeledElement: string | undefined,
```

Comment:
I would rename `modeledElement` to `modeledElementClass` or something like that.

Reply:
Renamed